python/lsst/sims/featureScheduler/utils/utils.py

```diff
@@ -897,22 +898,35 @@ def filter_count_ratios(target_maps):
 
 def run_info_table(observatory):
     """
-    Make a little table for recording the information of a run
+    Make a little table for recording the information about a run
     """
-    names = ['time', 'datetime', 'ymd', 'version', 'fingerprint', 'observatory_class', 'obs_finger']
-    types = [float, '', 'U20', '', 'U20', '', 'U20', '', 'U50', '', 'U20', '', 'U50']
-    result = np.zeros(1, dtype=list(zip(names, types)))
-    result['time'] = np.float(time.time())
+
+    observatory_info = observatory.get_info()
+
+    n_feature_entries = 4
+
+    names = ['Parameter', 'Value']
+    dtypes = ['|U60', '|U70']
+    result = np.zeros(observatory_info[:, 0].size + n_feature_entries,
+                      dtype=list(zip(names, dtypes)))
+
+    # Fill in info about the run
+    result[0]['Parameter'] = 'Date, ymd'
     now = datetime.datetime.now()
-    result['ymd'] = '%i, %i, %i' % (now.year, now.month, now.day)
-    result['version'] = version.__version__
-    result['fingerprint'] = version.__fingerprint__
-    result['observatory_class'] = observatory.__class__.__name__
-
-    try:
-        result['obs_finger'] = observatory.version.__fingerprint__
-    except:
-        pass
+    result[0]['Value'] = '%i, %i, %i' % (now.year, now.month, now.day)
+
+    result[1]['Parameter'] = 'hostname'
+    result[1]['Value'] = socket.gethostname()
```

Comment:
This may be better to get from an environment variable, and then have FBS set the OPSIM_HOSTNAME at the setup time. 
One reason for doing that is in case the hostname has characters not well understood by the database. (which may be less of an issue now than it used to be .. but astro-lsst-01 used to defeat sqlite and had to be replaced by astro_lsst_01 via OPSIM_HOSTNAME). 

I'm not sure if running on a cluster makes using OPSIM_HOSTNAME more desireable as well .. depends on if the hostname is still unique on all nodes. 

What happens to the hostname on a docker image (I'm actually not sure .. are all the 'hostnames' the same in that case?)

Reply:
Looks like on hyak `socket` returns things like `n2245.hyak.local`, so that seems fine. 

For docker, the hostname defaults to the container's ID, but you can set it to anything with `--hostname` when you run the container. 

I'm just treating it as a string, so sqlite shouldn't have issues with dashes, etc.

Reply:
The string vs. not wasn't the problem, I don't think. But it's true that sqlite doesn't seem to have issues with '-' in strings/varchar fields anymore, so I am guessing it'll be fine. 
Need to document the issue with docker, but agreed that setting --hostname vs. setting a OPSIM_HOSTNAME env variable is about equivalent.